src/App.js

```diff
@@ -53,13 +53,13 @@ function App() {
     fetch(url)
       .then((response) => response.json())
       .then(async (data) => {
-        //setFetchingData(false);
+        data.shift();
```

Comment:
we only want to do this when all the results are loaded I think? if you search/use geo, the first result isn't blank